app/models/daos/UserDAO.scala

```diff
@@ -41,35 +43,73 @@ class UserDAO @Inject()(dbapi: DBApi)(implicit ec: DatabaseExecutionContext) {
   def save(user: User): Future[User] = Future {
     val userId = db.withConnection { implicit conn =>
       user match {
-        case User(_, first, credentialId, last, userType, email) =>
+        case User(_, first, googleKey, last, userType, email) =>
           SQL(s"""
-            INSERT INTO USERS
+            INSERT INTO users
               (
                 first,
                 last,
                 email,
                 type,
-                credential_id
+                google_key
               )
             VALUES
               (
                 {first},
                 {last},
                 {email},
                 {userType},
-                {credentialId}
+                {googleKey}
               )
           """)
             .on(
               "first" -> first,
               "last" -> last,
               "email" -> email,
               "userType" -> userType,
-              "credentialId" -> credentialId).executeInsert()
+              "googleKey" -> googleKey).executeInsert()
+      }
+    }
+    user.copy(userID = userId.get.toInt)
+  }
+
+  def update(user: User): Future[User] = Future {
+    val userId = db.withConnection { implicit conn =>
+      user match {
+        case User(userId, first, googleKey, last, userType, email) =>
+          SQL(s"""
+            UPDATE users
+              SET
+                first = {first},
+                last = {last},
+                type = {userType},
+                google_key = {googleKey}
+              WHERE
+                id = {userId}
+          """)
+            .on(
```

Comment:
Worth creating `anorm.ToParameterList` instance for `User` so can use `.bind` like with `Job`?